addons/website_livechat/static/src/web/livechat_channel_info_list_patch.xml

```diff
@@ -3,6 +3,17 @@
     <t t-name="website_livechat.LivechatChannelInfoList" t-inherit="im_livechat.LivechatChannelInfoList" t-inherit-mode="extension">
         <xpath expr="//t[@t-name='extra_infos']" position="inside">
             <t t-set="visitor" t-value="props.thread.livechat_visitor_id"/>
+            <div t-if="visitor and (visitor.lang_id or visitor.country_id)">
+                <h3 class="pt-3">Country &amp; Language</h3>
+                <div class="d-flex bg-inherit gap-1">
+                    <span t-if="visitor.country_id">
+                        <img class="o_country_flag" t-att-src="visitor.country_id.flagUrl" t-att-title="visitor.country_id.code or visitor.country_id.name" aria-label="Country"/>
+                    </span>
+                    <span title="Language" t-if="visitor.lang_id">
+                        <t t-out="visitor.lang_id.name"/>
+                    </span>
```

Comment:
Nitpick:
- `t-if` as 1st attribute as this is the most important attribute
- `t-out` can be moved as attribute of `<span>, so 1-line for all this in template (shorter is better)